obs-packaging/linux-container/kata-linux-container.spec-template

```diff
@@ -82,6 +82,11 @@ BuildKernel() {
     find kata-kernel-configs -name "${kernelArch}_kata_kvm_*" -exec cp {} .config \;
 	[ -f .config ] || (echo "ERROR: cannot find the kernel config file for the ${kernelArch} architecture"; exit 1)
 
+    %if 0%{?rhel_version} || 0%{?suse_version}
+    # Not all OBS distros has updated gcc.
```

Comment:
This appears to be undoing a security feature so can you be more specific here as to why this is required? Are we really saying that the latest official RHEL and SuSE compilers cannot handle this?

Reply:
@jcvenegas yes, those are not new enough to support that feature I prefer to disable at distro level rather than config level.

Reply:
I'm finding it difficult to believe that RH and SuSE haven't patched for this yet. Also, I've just checked on a Centos test box:

```sh
$ cat /etc/centos-release
CentOS Linux release 7.6.1810 (Core)
$ sudo cat /sys/devices/system/cpu/vulnerabilities/spectre_v2
Mitigation: Full retpoline
$ grep CONFIG_RETPOLINE=y /boot/config-`uname -r`
CONFIG_RETPOLINE=y
```

Reply:
Any input on this @marcov?

Reply:
I quickly checked on the SLES 12.3 and 15, both have in `CONFIG_RETPOLINE=y` in their `/proc/config.gz`